### ISSUE TITLE
DDFFNet for mDFF

### ISSUE THREAD

**Issue by nancibles** [MaximilianStaab/mDFF#4]
Thanks to your help with the pre-processing, I was able get the pointcloud data and the focal stacks packaged for use with DDFFNet. I was able to replicate the results of the DDFF 12-Scene dataset with DDFFNet and need to use it with depth, not disparity, maps now. Unfortunately, I only had a small dataset captured before the city lockdown so I am now trying to the mDFF dataset in addition to my own dataset to train the network since they are similar enough. The link to the download the cropped and clipped mDFF dataset did not work for me so I ran the pack_to_h5.py script on the original data to get the h5 file.

Training the network was unsuccessful, even without using my own dataset. When training, the loss seems very high and after training around 15 epoches, it's clear that it is not converging. I'm wondering if there were any changes aside from the clipping range to DDFFNet that were made to adapt DDFFNet (ddff-pytorch repository) to work with mDFF dataset? 

Sorry for bothering you again but I very much appreciate all of your help thus far.
